client/my-sites/plans-v2/products-column/index.tsx

```diff
@@ -55,6 +56,7 @@ const ProductComponent = ( {
 		features={ { items: [] } }
 		discountedPrice={ product.discountCost }
 		originalPrice={ product.cost || 0 }
+		withStartingPrice={ slugIsSelectorProductSlug( product.productSlug ) }
```

Comment:
The `SelectorProduct` type has a `subtypes` key for this reason. I would suggest this instead:
```suggestion
		withStartingPrice={ product.subtypes.length > 0 }
```

Reply:
Much better, thanks!